src/Tooltip.svelte

```diff
@@ -1,6 +1,6 @@
 <script>
   import { onDestroy, onMount } from 'svelte';
-  import { createPopper } from '@popperjs/core/dist/esm/popper';
+  import { createPopper } from '@popperjs/core';
```

Comment:
I'm not sure if we can change this, popper has this annoying dependency on node process and this was causing Sveltestrap to break in the svelte.dev repl

Please see this issue: https://github.com/bestguy/sveltestrap/issues/250